k8s.go

```diff
@@ -18,6 +18,8 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/yaml"
+
+	convertpkg "github.com/istio-ecosystem/security-policy-migrate/converter"
```

Comment:
Better to modify the variable `converter` instead of renaming the imported package.

Reply:
Done